ast/policy.go

```diff
@@ -130,6 +130,15 @@ type (
 		Location *Location
 	}
 
+	// Annotation contains code annotation
+	Annotation interface{ annotationMaker() }
+
+	// SchemaAnnotation contains information about a schema
+	SchemaAnnotation struct {
```

Comment:
💭 Do we need json tags on this struct?

Reply:
I think we should consider this type in context of #2180. Let's assume:

* Annotations could be defined at the package or rule level
* Annotations may contain more than just schema, e.g., links, examples, human-readable text, custom sections, etc.